src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.scss

```diff
@@ -9,6 +9,10 @@
   padding: 75px 20px 10px 20px;
   height: 100%; // may need to change this later
   width: 100%;
+  max-width: 100vw;
```

Comment:
This (and the previous) lines are probably useless. geneView is just a div whose width is controlled by the width of the main container of StandardAppLayout

Reply:
Removed it